proto/src/determined/api/v1/user.proto

```diff
@@ -39,6 +39,12 @@ message GetUsersRequest {
 
   // Filter by username or display name.
   string name = 5;
+  // Filter by status.
+  optional bool active = 6;
+  // Filter by roles.
+  optional bool admin = 7;
+  // Filter by rabc cluster scoped roles.
+  optional int32 roleId = 8;
```

Comment:
Can we make the EE only endpoint to be exactly the same as `GetUsers` endpoint of OSS, only to allow one extra field `roleId`? That would be easier from frontend point of view, since user list are not coming from two separate endpoints.

Reply:
Yeah I think that would be fine